### ISSUE TITLE
System check missing check for extension "zip"

### ISSUE THREAD

**Issue by SunBlack** [contao/contao-manager#835]
The installation didn't completed with xampp as `extension=zip` also needs to be activated in the `php.ini`. I think the system check should check this.

**Comment by aschempp** [comment]
any idea what the extension was used for?

@Toflar am I right that we either need the ZIP extension or CLI ZIP support for Composer? So the Manager should probably mimic that check?

**Comment by Toflar** [comment]
Composer does not require the `zip` extension. Composer tries to find the fastest command available (`unzip`, `7z` etc.) depending on the platform and only uses the PHP `ZipArchive` as fallback if none of the faster variants are available. 
It might be that in this case absolutely no zip variant was available but we cannot tell because @SunBlack did not provide any error message.

**Comment by SunBlack** [comment]
Sorry for not pasting the error message. But found it on another issue from me:
![398719958-d785cbbf-2ba0-4929-866f-f801f970ef9c](https://github.com/user-attachments/assets/2da381e7-1985-4323-b638-255e864bfc06)

**Comment by Toflar** [comment]
Yes, this is exactly the error message. There is no zip utility available. But imho checking for the `zip` extension is not correct because again, it's not strictly required. You should provide another utility for best performance in Composer anyway.
We could, however, run `composer diagnose` - it would fail if something is not okay. The output on my computer for the archive related stuff looks like this:
```
zip: extension present, unzip present, 7-Zip not available
```

It also checks for other stuff like OpenSSL, curl, php version and so on. So this might be an approach to fix this.

**Comment by aschempp** [comment]
Unfortunately, the `composer diagnose` command is not accessible and uses only private method. I have added a ZIP check in 30169017d98d632545f22c6e624df0678451ff40 for now.
